### DISCUSSION TITLE
Why ComputeRouteMatrixResponse not present?

### DISCUSSION THREAD

**Discussion by assizkii** [googleapis/googleapis#791] [Q&A]
ComputeRouteMatrixRequest is in the [documentation](https://cloud.google.com/go/docs/reference/cloud.google.com/go/maps/latest/routing/apiv2#cloud_google_com_go_maps_routing_apiv2_RoutesClient_ComputeRouteMatrix), but the protofiles don't contain ComputeRouteMatrixResponse, why? I want to use Route Matrix for some business cases.

**Comment by Cutebarb** [comment]
- [ ]
